examples/polygon-p2p/src/main.rs

```diff
@@ -72,7 +72,7 @@ async fn main() {
     while let Some(evt) = events.next().await {
         // For the sake of the example we only print the session established event
         // with the chain specific details
-        if let NetworkEvent::SessionEstablished { status, client_version, .. } = evt {
+        if let Ok(NetworkEvent::SessionEstablished { status, client_version, .. }) = evt {
```

Comment:
let's log the error here, at least with `trace` log level. we may want to add proper error handling here, but as a separate issue.